extension/scripts/features/faction-quick-items/ttFactionQuickItems.js

```diff
@@ -458,7 +458,7 @@
 
 				item.removeEventListener("click", onItemClickQuickEdit);
 			}
-			for (const refill of document.findAll("#armoury-points .give[data-role='give']")) {
+			for (const refill of document.findAll("#armoury-points .give[data-role='give'], .give[data-role='refill']")) {

```

Comment:
Include the `#armoury-points` selector with the new part as well.

Reply:
Done